test/Atmos/EDMF/closures/entr_detr.jl

```diff
@@ -64,7 +64,7 @@ function entr_detr(
     lim_amp = entr.lim_amp
     w_min = entr.w_min
     # precompute vars
-    w_up_i = up[i].ρaw / up[i].ρa
+    w_up_i = up[i].ρa < eps(FT) ? FT(0) : up[i].ρaw / up[i].ρa
```

Comment:
I think it might be better not to compute enter/detr at all if a=0, its unneeded computation